pandas/core/indexing.py

```diff
@@ -1905,17 +1900,14 @@ def _validate_key(self, key, axis: int):
 
             # check that the key has a numeric dtype
             if not is_numeric_dtype(arr.dtype):
-                raise IndexError(
-                    ".iloc requires numeric indexers, got {arr}".format(arr=arr)
-                )
+                raise IndexError(f".iloc requires numeric indexers, got {arr}")
 
             # check that the key does not exceed the maximum size of the index
             if len(arr) and (arr.max() >= len_axis or arr.min() < -len_axis):
                 raise IndexError("positional indexers are out-of-bounds")
         else:
             raise ValueError(
-                "Can only index by location with "
-                "a [{types}]".format(types=self._valid_types)
+                "Can only index by location with " f"a [{self._valid_types}]"
```

Comment:
extra space crept in, also down on 2059 (im going to stop pointing those out now)

Reply:
My fault. Missed them.